extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -132,38 +133,164 @@
+	'userprofilepage-add-question' => 'Currently not in use!',
+	'userprofilepage-question-save' => 'Currently not in use!',
+	'userprofilepage-question-list-title' => 'Currently not in use!
+	Parameters:
 * $1 is the number of the current question
 * $2 is the total number of questions in the interview,',
-	'user-identity-box-edit' => 'This is the label of a link leading to profile editing mode. Should be used as a verb phrase.',
-	'user-identity-box-aka-label' => "Parameters:
+	'userprofilepage-interview-section-title' => 'Currently not in use!',
+	'userprofilepage-invalid-user' => 'Error message, appears when user has not sufficient rights to edit user profile page or is not logged in.',
+	'userprofilepage-avatar-error-type' => 'Error message, appears when user uploads avatar picture file in incorrect format (not supported mime type),
+	Parameters
+* $1 is list of supported mime types',
+	'userprofilepage-avatar-error-resolution' => 'Error message, appears when user tries to upload to big avatar file. This message contains information that maximum avatar resolution is 2000x2000px',
+	'user-identity-box-edit' => 'Label for link that opens user profile edit modal.',
+	'user-identity-box-wikia-contributor' => 'Anonymous user "real name". Displayed when we display user profile page for IP address after "aka" phrase.',
+	'user-identity-box-edit-avatar' => 'Label for link which opens user avatar picture edit modal.',
+	'user-identity-box-delete-avatar' => 'Label for link which deletes current user avatar picture, after user click this link confirmation message is displayed.',
+	'user-identity-box-avatar-upload-avatar' => 'Label for form file input on avatar picture edit modal.',
+	'user-identity-box-avatar-fb-import-avatar' => 'Currently not in use!',
+	'user-identity-box-avatar-choose-avatar' => 'Label that encourages user to choose one of default avatar pictures on avatar picture edit modal.',
+	'user-identity-box-avatar-save' => "Label for save button on avatar picture edit modal.",
+	'user-identity-box-avatar-cancel' => 'Label for cancel button on avatar picture edit modal. Click on this button close modal or displays confirmation message if any changes where already made.',
+	'user-identity-box-avatar-anon-user-error' => 'Error message that is displayed when anon user tries to change his avatar. It tells that user needs to login before he change his avatar.',
+	'user-identity-box-hide-edits-wikis' => 'Label for checkbox on user profile edit modal. When this field is checked user favorite wikis are not displayed on user profile page.',
+	'user-identity-box-banned-from-chat' => 'Label for tag displayed just after user login on user profile page which tell that used is banned for using Wikia chat.',
+
+	'user-identity-box-aka-label' => "Message that is displayed below user name on user profile page that shows user real name. In English there is 'aka - also known as' in HTML span tag (it's required for styling).
```

Comment:
HTML in messages will be removed in: https://wikia-inc.atlassian.net/browse/SOC-417